share/rocm/cmake/ROCMCreatePackage.cmake

```diff
@@ -132,6 +132,46 @@ macro(rocm_package_add_rocm_core_dependency)
     endif()
 endmacro()
 
+function(rocm_parse_python_syspath)
+    set(PYTHON_SITE_PACKAGES
+        "/usr/lib/python3/dist-packages;/usr/lib/python2.7/dist-packages"
+        CACHE STRING "The site packages used for packaging")
+    #Group the statements to function
+    file(APPEND ${PROJECT_BINARY_DIR}/debian/postinst
+        "
+        set_libdir(){
+    ")
+    file(APPEND ${PROJECT_BINARY_DIR}/debian/prerm
+        "
+        rm_libdir(){
+    ")
+    foreach(PYTHON_SITE ${PYTHON_SITE_PACKAGES})
+        file(
+            APPEND ${PROJECT_BINARY_DIR}/debian/postinst
+            "
+            mkdir -p ${PYTHON_SITE}
+            echo \"${LIB_DIR}\" > ${PYTHON_SITE}/${PARSE_NAME}.pth
```

Comment:
I'm wary of using `PARSE_NAME` here, if the `rocm_create_package` variables ever change their names this would be very easy to miss as it's in a different scope. Consider passing this in as a parameter?

Reply:
Thanks @lawruble13 for the comments. Will work on that

Reply:
Passed PARSE_NAME and LIB_DIR as function arguments